release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -91,7 +91,9 @@ You can deploy an {product-title} cluster to multiple vSphere datacenters or reg
 
 [IMPORTANT]
 ====
-The VMware vSphere region and zone enablement feature is only available with a newly installed cluster, because this feature requires the Container Storage Interface (CSI) storage class. A cluster that was upgraded from a previous release does not include the required CSI storage class.
+The VMware vSphere region and zone enablement feature is only available with a newly installed cluster, because this feature requires the vSphere Container Storage Interface (CSI) driver as the default storage driver in the cluster.
+
+A cluster that was upgraded from a previous release defaults to using the in-tree vSphere driver, so you must enable CSI automatic migration for the cluster. You can then configure multiple regions and zones for the upgraded cluster. 
```

Comment:
```suggestion
The VMware vSphere region and zone enablement feature is only available with a newly installed cluster because it requires the vSphere Container Storage Interface (CSI) driver as the default storage driver in the cluster.

A cluster that was upgraded from a previous release defaults to the in-tree vSphere driver. As a result, you must enable CSI automatic migration for the cluster to use the feature. You can then configure multiple regions and zones for the upgraded cluster. 
```

Suggestion.

Reply:
"It" might be misconstrued here, as it can be a vague referent: the cluster or the feature. I do not want our customers thinking that the CSI driver being set as the default storage driver is essential for all newly installed cluster on vSphere.

Reply:
How about:

> TO use this feature on a newly installed cluster, you must enable CSI automatic migration for the cluster.

Reply:
Fine by me.